testdata/deponixml/noark5-enkel/arkivstruktur.xml

```diff
@@ -74,8 +75,7 @@
         <opprettetAv>Arkivnotat OpprettetAv</opprettetAv>
         <arkivertDato>1863-10-10T00:00:00Z</arkivertDato>
         <arkivertAv>Arkivnotat Arkivertav</arkivertAv>
-	<part>
-          <partID>Sakspart2</partID>
```

Comment:
Similarly to krissreferanse, could not ignore partID so I removed it just to continue.